pages/library/_viewId.vue

```diff
@@ -182,10 +181,11 @@ export default Vue.extend({
     ...mapActions('page', ['setPageTitle']),
     async filterMedia(options: ItemsApiGetItemsRequest) {
```

Comment:
This entire function won't exist when #267 is merged, not sure it's worth fixing here.

Reply:
Removed this commit